### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by burtonemily**
Updating tests to include onboarding changes

### CODE REVIEW

File: tests/automation/utilities/utils.ts [hunk 1]
```diff
@@ -395,6 +405,28 @@ export async function hasElementPoppedUpThatShouldnt(
...
+  selector: string,
+  text?: string,
+) {
+  const builtSelector = !text
+    ? `css=[${strategy}=${selector}]`
+    : `css=[${strategy}=${selector}]:has-text("${text.replace(/"/g, '\\"')}")`;
+
+  const fakeError = `Element ${selector} does not exist`;
+  const elVisible = await window.isVisible(builtSelector);
+  if (elVisible === false) {
```
Comment (Bilb) [inline tests/automation/utilities/utils.ts:423]: if(!elVisible)

File: tests/automation/utilities/utils.ts [hunk 2]
```diff
@@ -395,6 +405,28 @@ export async function hasElementPoppedUpThatShouldnt(
...
+    console.log(fakeError);
+    return;
+  } else {
```
Comment (Bilb) [inline tests/automation/utilities/utils.ts:426]: no need for else after a return

File: tests/automation/setup/recovery_using_seed.ts [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+  await clickOnTestIdWithText(window, 'continue-button');
+  await waitForLoadingAnimationToFinish(window, 'loading-animation');
+  const displayName = await doesElementExist(
+    window,
+    'data-testid',
+    'display-name-input',
+  );
+  if (displayName) {
+    throw new Error(`Display name was not found when restoring from seed`);
+  } else {
```
Comment (Bilb) [inline tests/automation/setup/recovery_using_seed.ts:22]: no need for else

File: tests/automation/group_testing.spec.ts [hunk 4]
```diff
@@ -143,8 +142,9 @@ test_group_Alice_1W_Bob_1W_Charlie_1W(
     await clickOnTestIdWithText(aliceWindow1, 'edit-group-name');
     await typeIntoInput(aliceWindow1, 'group-name-input', '     ');
     await aliceWindow1.keyboard.press('Enter');
-    const errorMessage = aliceWindow1.locator('.error-message');
-    await expect(errorMessage).toContainText('Please enter a group name');
+    await waitForMatchingText(aliceWindow1, 'Please enter a group name');
+    // const errorMessage = aliceWindow1.locator('.error-message');
+    // await expect(errorMessage).toContainText('Please enter a group name');
```
Comment (Bilb) [inline tests/automation/group_testing.spec.ts:147]: to be removed?

Comment (burtonemily) [reply tests/automation/group_testing.spec.ts:147]: No, i've put adding in a data-test id into the backlog for an error message here, so i thought i would just leave it for future...

File: .drone.jsonnet [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
-  name: 'session-integration-tests-' + shard_number,
-  platform: { arch: 'amd64' },
-  // depends_on: ['restore-cache'],
-  steps: [
+// local playwright_shard(shard_number) = {
+//   kind: 'pipeline',
+//   type: 'docker',
+//   name: 'session-integration-tests-' + shard_number,
+//   platform: { arch: 'amd64' },
+//   // depends_on: ['restore-cache'],
```
Comment (Bilb) [inline .drone.jsonnet:6]: you can remove this file altogether. We won't use drone, no matter what we do.,

**Review by Bilb — CHANGES_REQUESTED** [review]

**Review by Bilb — APPROVED** [review]

**Pull request merged**